reports/report.json

```diff
@@ -286,29 +286,23 @@
             "src": "1026:80",
             "src_char": "1026:80"
           },
-          {
-            "contract_path": "src/uniswap/UniswapV2Swapper.sol",
-            "line_no": 28,
-            "src": "1116:67",
-            "src_char": "1116:67"
-          },
           {
             "contract_path": "src/uniswap/UniswapV2Swapper.sol",
             "line_no": 31,
-            "src": "1261:112",
-            "src_char": "1261:112"
+            "src": "1278:112",
+            "src_char": "1278:112"
           },
           {
             "contract_path": "src/uniswap/UniswapV2Swapper.sol",
             "line_no": 32,
-            "src": "1383:99",
-            "src_char": "1383:99"
+            "src": "1400:99",
+            "src_char": "1400:99"
           },
           {
             "contract_path": "src/uniswap/UniswapV2Swapper.sol",
             "line_no": 33,
-            "src": "1492:109",
-            "src_char": "1492:109"
+            "src": "1509:109",
+            "src_char": "1509:109"
           },
```

Comment:
WHy have these changed ?

Reply:
Ahh now I see, it's because I added the comments to ignore, which changes the offset of existing nodes. All cool !